cmd/hub/mcscontroller/main.go

```diff
@@ -73,15 +78,15 @@ func main() {
 	flag.Parse()
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
-	if namespace == "" {
+	if namespace == corev1.NamespaceAll {
 		ns, err := getWatchedNamespace()
 		if err != nil {
 			setupLog.Error(err, "Failed to get watched namespace from environment")
```

Comment:
dont you want to exit here? or is it just a warning?

Reply:
we should exit if the namespace is not specified in CLI or environment.
We won't run with cluster scope